LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -227,13 +258,7 @@
 	<owl:ObjectProperty rdf:about="&fibo-loan-typ-mtg;hasMIGProvider">
 		<rdfs:label xml:lang="en">has m i g provider</rdfs:label>
```

Comment:
Use MIG not m i g

Reply:
removed the property - used hasGuarantor